fdb-record-layer-core/src/main/proto/record_query_plan.proto

```diff
@@ -1370,6 +1348,15 @@ message PRecordQueryIndexPlan {
   optional PQueryPlanConstraint constraint = 9;
 }
 
+//
+// PTempTableScanPlan
+//
+message PTempTableScanPlan {
+  optional PValue temp_table = 1;
```

Comment:
should be the value

Reply:
renamed to `temp_table_reference_value`.